src/library_webgl.js

```diff
@@ -3601,6 +3601,49 @@ var LibraryGL = {
     GLctx.sampleCoverage(value, !!invert);
   },
 
+  glMultiDrawArraysANGLE: function(mode, firsts, counts, drawcount) {
+    GLctx.multiDrawWebgl['multiDrawArraysWEBGL'](
+      mode,
+      HEAP32,
+      firsts >> 2,
+      HEAP32,
+      counts >> 2,
+      drawcount);
+  },
+  glMultiDrawArraysInstancedANGLE: function(mode, firsts, counts, instanceCounts, drawcount) {
+    GLctx.multiDrawWebgl['multiDrawArraysInstancedWEBGL'](
+      mode,
+      HEAP32,
+      firsts >> 2,
+      HEAP32,
+      counts >> 2,
+      HEAP32,
+      instanceCounts >> 2,
+      drawcount);
+  },
+  glMultiDrawElementsANGLE: function(mode, counts, type, offsets, drawcount) {
+    GLctx.multiDrawWebgl['multiDrawElementsWEBGL'](
+      mode,
+      HEAP32,
+      counts >> 2,
+      type,
+      HEAP32,
+      offsets >> 2,
+      drawcount);
+  },
+  glMultiDrawElementsInstancedANGLE: function(mode, counts, type, offsets, instanceCounts, drawcount) {
+    GLctx.multiDrawWebgl['multiDrawElementsInstancedWEBGL'](
+      mode,
+      HEAP32,
+      counts >> 2,
+      type,
+      HEAP32,
+      offsets >> 2,
+      HEAP32,
+      instanceCounts >> 2,
+      drawcount);
+  },
+
```

Comment:
Let's name these functions according to the extension spec, that is the convention we have taken with other extensions as well. So `glMultiDrawElementsInstancedWEBGL` and so forth.

Then, to enable portability, add aliases
```js
glMultiDrawElementsInstancedWEBGL__sig: 'viiiiii',
glMultiDrawElementsInstancedANGLE: 'glMultiDrawElementsInstancedWEBGL'
```
That will cause the functions to be exposed under both names.